device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/port_config.ini

```diff
@@ -1,57 +1,57 @@
-Ethernet28      28,29,30,31            100000       etp8        8
-Ethernet32      32,33,34,35            100000       etp9        9
-Ethernet36      36,37,38,39            100000       etp10       10
-Ethernet40      40,41                  50000        etp11a      11
-Ethernet42      42,43                  50000        etp11b      11
-Ethernet44      44,45                  50000        etp12a      12
-Ethernet46      46,47                  50000        etp12b      12
-Ethernet48      48,49                  50000        etp13a      13
-Ethernet50      50,51                  50000        etp13b      13
-Ethernet52      52,53                  50000        etp14a      14
-Ethernet54      54,55                  50000        etp14b      14
-Ethernet56      56,57                  50000        etp15a      15
-Ethernet58      58,59                  50000        etp15b      15
-Ethernet60      60,61                  50000        etp16a      16
-Ethernet62      62,63                  50000        etp16b      16
-Ethernet64      64,65                  50000        etp17a      17
-Ethernet66      66,67                  50000        etp17b      17
-Ethernet68      68,69                  50000        etp18a      18
-Ethernet70      70,71                  50000        etp18b      18
-Ethernet72      72,73                  50000        etp19a      19
-Ethernet74      74,75                  50000        etp19b      19
-Ethernet76      76,77                  50000        etp20a      20
-Ethernet78      78,79                  50000        etp20b      20
-Ethernet80      80,81                  50000        etp21a      21
-Ethernet82      82,83                  50000        etp21b      21
-Ethernet84      84,85                  50000        etp22a      22
-Ethernet86      86,87                  50000        etp22b      22
-Ethernet88      88,89,90,91           100000        etp23       23
-Ethernet92      92,93,94,95           100000        etp24       24
-Ethernet96      96,97,98,99           100000        etp25       25
-Ethernet100     100,101,102,103       100000        etp26       26
-Ethernet104     104,105                50000        etp27a      27
-Ethernet106     106,107                50000        etp27b      27
-Ethernet108     108,109                50000        etp28a      28
-Ethernet110     110,111                50000        etp28b      28
-Ethernet112     112,113                50000        etp29a      29
-Ethernet114     114,115                50000        etp29b      29
-Ethernet116     116,117                50000        etp30a      30
-Ethernet118     118,119                50000        etp30b      30
-Ethernet120     120,121                50000        etp31a      31
-Ethernet122     122,123                50000        etp31b      31
-Ethernet124     124,125                50000        etp32a      32
-Ethernet126     126,127                50000        etp32b      32
+Ethernet0       0,1                    50000        etp1a       0
```

Comment:
Can you check if you need to make a corresponding change here ? https://github.com/Azure/sonic-mgmt/blob/master/ansible/library/minigraph_facts.py#L476

Reply:
Hi Prince,
The purpose of this code is to generate a mapping from port alias to name. It doesn't reference the "port index" column. So no need to change it.